magenta/music/musicxml_parser.py

```diff
@@ -358,6 +361,8 @@ def _parse(self, xml_part, score_parts):
     xml_measures = xml_part.findall('measure')
     for child in xml_measures:
       measure = Measure(child, self._state)
+      # Update the time signature if a partial or pickup measure
+      measure.check_time_signature()
```

Comment:
rather than requiring check_time_signature to be called after Measure is constructed, should it be called by the Measure constructor after parse? If so, it should be renamed to `_check_time_signature`.